yalla-coop/tempo
# 602 earngroups for individ programme

#### Issues this PR relates to
#602 

how to test : 
  - sign up as staff member
  - go to programme tab in navbar
  - click on any programme in the table 
  - click on earn groups card this will lead you to `programmeEarnGroups` Page and it's link : http://localhost:3000/admin/programmes/:id/earn-groups
  - then to see invite page click on `add earn group` button and it's link: http://localhost:3000/admin/programmes/:id/invite
  
---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs
